src/ui/include/nqqsettings.h

```diff
@@ -81,6 +81,7 @@ class NqqSettings {
         NQQ_SETTING(RecentDocuments,                QList<QVariant>, QList<QVariant>())
 
         NQQ_SETTING(NotepadqqVersion,               QString,    QString())
+        NQQ_SETTING(SetSmartIndentation,            bool,       true)
```

Comment:
@qcerasmus @JuBan1  wouldn't "SmartIndentation" be better instead of "SetSmartIndentation"?

Reply:
That came to my mind as well but after looking at the other settings it turns out we actually have a lot of those "verb-style" names. "RememberTabsOnExit", "ShowSpaces", "OverrideFontFamily", "CheckVersionAtStartUp". So it's actually not out of the ordinary.

Reply:
true, however this particular case creates weird/confusing things down in the code:

`m_settings.General.getSetSmartIndentation()`, `m_settings.General.setSetSmartIndentation()`

I think it's better to change it now before it goes into the users settings file, I can do it